hlxsites/merative2
# fix(quick-links): update patterns and add theme tokens

## Issue

Fixes - https://jira.sdlc.merative.com/browse/MERATIVE-821

## Description

**Changed**

- This PR is a follow-up to https://github.com/hlxsites/merative2/pull/318
- Adds the updated SVG patterns for the super 6 solutions
- Adds theme variable tokens for the brand for code consolidation and maintainability 

## Design Specs

- Figma Link - No design specs


## Preview Links

### Curam
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/quick-links/curam
- After (Changes from this PR): https://fix-quick-links-themes--merative2--proeung.hlx.page/quick-links/curam

### Health Insights
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/quick-links/health-insights
- After (Changes from this PR): https://fix-quick-links-themes--merative2--proeung.hlx.page/quick-links/health-insights

### MarketScan
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/quick-links/marketscan
- After (Changes from this PR): https://fix-quick-links-themes--merative2--proeung.hlx.page/quick-links/marketscan

### Merge
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/quick-links/merge
- After (Changes from this PR): https://fix-quick-links-themes--merative2--proeung.hlx.page/quick-links/merge

### Micromedex
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/quick-links/micromedex
- After (Changes from this PR): https://fix-quick-links-themes--merative2--proeung.hlx.page/quick-links/micromedex

### Zelta
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/quick-links/zelta
- After (Changes from this PR): https://fix-quick-links-themes--merative2--proeung.hlx.page/quick-links/zelta


  
## Testing Instruction

- Visit the preview links above and ensure that CTA button color and patterns are rendering correctly.
